service/history/workflow/mutable_state_impl.go

```diff
@@ -555,6 +555,7 @@ func NewMutableStateInChain(
 
 	// carry over necessary fields from current mutable state
 	newMutableState.executionInfo.WorkflowExecutionTimerTaskStatus = currentMutableState.GetExecutionInfo().WorkflowExecutionTimerTaskStatus
+	newMutableState.executionInfo.ChildrenInitializedPostResetPoint = currentMutableState.GetExecutionInfo().ChildrenInitializedPostResetPoint
```

Comment:
This should be ok since entries in `currentMutableState.GetExecutionInfo().ChildrenInitializedPostResetPoint` will not be updated beyond this point. If there are any possibility of that happening, we should deep copy the map contents here. Please let me know.